presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkModule.java

```diff
@@ -457,6 +461,9 @@ protected void setup(Binder binder)
         binder.bind(SpillerStats.class).in(Scopes.SINGLETON);
         binder.bind(StandaloneSpillerFactory.class).to(TempStorageStandaloneSpillerFactory.class).in(Scopes.SINGLETON);
 
+        // Broadcast file storage.
+        binder.bind(BroadcastStorage.class).to(LocalBroadcastStorage.class).in(Scopes.SINGLETON);
```

Comment:
It seems strange to hard-code LocalBroadcastStorage. Shouldn't this be driven by configuration?

Reply:
@singcha  Maybe we can add an `AbstractBroadcastStorage` which throws an `UnsupportedOperation` exception to indicate that the users need to implement their own impl before using this codepath. Also assuming we will bind to `LocalBroadcastStorage` for test modules

Reply:
Not applicable anymore